src/test/regress/Makefile

```diff
@@ -86,6 +86,9 @@ endif
 
 isolation_test_files=$(generated_isolation_files) create-symbolic-link
 
+CITUS_VALGRIND_LOG_FILE=citus_valgrind_test_log.txt
+COLUMNAR_VALGRIND_LOG_FILE=columnar_valgrind_test_log.txt
+
```

Comment:
Where was VALGRIND_LOG_FILE set before? If it came from the test env, perhaps we need these to come from the test env also?

Reply:
From 1da42f7a39fffb3ea52a74d761a6ebcf5eab8141 commit message:

>Previously, even if we were passing "VALGRIND_LOG_FILE" argument in
>Makefile, we were always writing to "valgrind_test_log.txt" since
>"VALGRIND_LOG_FILE" is actually empty in Makefile but
>"pg_regress_multi.pl" was setting log file to "valgrind_test_log.text"

Reply:
I can merge the pr if this makes sense to you @jeff-davis

Reply:
fixed by 10a42e3